composer.json

```diff
@@ -70,6 +70,7 @@
         "resque/php-resque": "1.3.6",
         "matomo/device-detector": "6.1.*",
         "dragonmantank/cron-expression": "3.3.2",
+        "influxdb/influxdb-php": "1.15.2",
         "phpmailer/phpmailer": "6.8.0",
         "chillerlan/php-qrcode": "4.3.4",
         "adhocore/jwt": "1.1.2",
```

Comment:
We forgot to remove statsd last time, let's make a note when we recover this feature for next release.